### PR TITLE
Updated use-a-frontend-library.md

### PR DESCRIPTION
**Pull request by Jassi10000**
Pertains to issue #184 
Updated the folder name of frontend part of the application from <code>litepy-react</code>  to  <code>myapp-react</code>

@pathange-s Sir it would be nice if you review and merge it

### CODE REVIEW

File: docs/how-to/use-a-frontend-library.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ You can add the following section to your configuration file for activating hot-
   "cli": {
     // --- other options
```
Comment (pathange-s) [inline docs/how-to/use-a-frontend-library.md:153]: Why is this removed ? This indicates other options of the cli in config.json.

Comment (Jassi10000) [reply docs/how-to/use-a-frontend-library.md:153]: Sir I haven't removed it
Its still present there

image

The red colour is signifying something else , I think you are confusing it with the light red colour which appears when we remove something 

See the original ones also has this
image

**Comment by Jassi10000** [comment]
> @Jassi10000 Don't call me Sir, you can call me Sai.
> 
> Cool if that line isn't removed. The fix lgtm. I can't approve the changes. Let the PR be open, one of other maintainers will approve and merge it.

Sure sir , no issues

**Review by shalithasuranga — APPROVED** [review]
Thank you very much for the contribution :+1:

File: docs/how-to/use-a-frontend-library.md (hunk 1 above)
Comment (shalithasuranga) [reply docs/how-to/use-a-frontend-library.md:153]: Seems like GitHub shows it in red color since inline-comments are not a part of the JSON spec ;)
